source/grid/grid_tools.cc

```diff
@@ -556,7 +556,7 @@ namespace GridTools
     class ShiftPoint
```

Comment:
I stumbled about this as well, but the term is used here in the "action" sense, i.e., the class implements an operation that "shifts the given point". It does not encapsulate a shift vector. So I think the class name is appropriate.